make.bash

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
```

Comment:
One small suggestion, let's template the `GO_VERSION` in the makefile as well for more static build.

Reply:
could you explain in more detail?   take  ```GO_VERISON``` to check  golang version in build command?